### PR TITLE
Updated levels + other small changes

### LINKED ISSUE #104: Think about which product belongs in which level (EST 1 IMP 100)

### PR DESCRIPTION
**Pull request by lukko**
- Added some more levels.
- Removed level with satisfaction type.
- Decreased the goals for the money levels.
- Level counter starts at 1.

- Lowered transport cost. _It's relatively high._
- Increased salesprice for apples and bananas. _These are the products which you start with, so for the feel-good idea I think the price can be higher._

Closes #104. And a bit of #108.

### CODE REVIEW

File: src/assets/levels.json [hunk 1]
```diff
@@ -1,11 +1,35 @@
 [
   {
     "type": "money",
+    "goal": 10010
```
Comment (N-Wouda) [inline src/assets/levels.json]: I think there are too many at this time. Not that it's a problem to have many levels, but perhaps make them a bit more substantial. Say 10250, 10500, 11000, 12000, 15000, 20000?

**Comment by N-Wouda** [comment]
Looks fine to me! Especially the price hikes, as that needs to happen. Right now the game is just _too hard_!

File: src/assets/levels.json (hunk 1 above)
Comment (lukko) [reply src/assets/levels.json]: My reasoning was that as a kid I would prefer to see that I'm progressing and levels kind of indicate that. I don't know about you, but I can't reach 11000 easily?

Comment (N-Wouda) [reply src/assets/levels.json]: You're not wrong, but that's why I think we should increase the difference between purchase and sales price. Simply make it easier to earn money!

**Comment by lukko** [comment]
I'm finished with this PR.

**Comment by lukko** [comment]
Now I'm finished :)

File: src/assets/products.json [hunk 2]
```diff
@@ -145,7 +145,7 @@
   },
   {
     "name": "Short",
-    "level": 2,
+    "level": 7,
```
Comment (N-Wouda) [inline src/assets/products.json:148]: Can these levels be reached now?

**Comment by N-Wouda** [comment]
I think only a check on the levels in the products remains and then this can be merged. I'm actively pushing to get most thing into the game (I'll work on it tomorrow too, just some finishing touches). Then we can have it shipped off and do other things for a while (although it might be fun to revisit "Serious games" in the future!)

**Comment by N-Wouda** [comment]
Merging this, I'll look into the levels myself.

**Pull request merged**